README.md

```diff
@@ -1 +1,60 @@
-# mp4pssh
\ No newline at end of file
+# MP4PSSH
+
+A parser to convert a Base64-encoded string representing an MP4 PSSH box into a POJO. More information about MP4 PSSH boxes can be found [here](https://w3c.github.io/encrypted-media/format-registry/initdata/cenc.html#common-system).
+
+## Installation
```

Comment:
add an import statement to show how someone can then use the library in their code base.